public/app/ui/pages/home/HomePage.less

```diff
@@ -0,0 +1,21 @@
+article#home {
```

Comment:
Added the wrapper of `article#home` which is the standard, to use the outermost container element of the template as the prefix of all the style rules.